deploy/kubei.yaml

```diff
@@ -38,117 +38,66 @@ roleRef:
   kind: ClusterRole
   name: kubei
 ---
-apiVersion: v1
-kind: Service
-metadata:
-  name: clair
-  namespace: kubei
-  labels:
-    app: clair
-spec:
-  type: ClusterIP
-  ports:
-  - port: 6060
-    protocol: TCP
-    name: apiserver
-  - port: 6061
-    protocol: TCP
-    name: health
-  selector:
-    app: clair
----
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: clair
+  name: grype-server
```

Comment:
can we follow least privilege best practices and use
```
        securityContext:
          capabilities:
            drop:
            - ALL
          runAsNonRoot: true
          runAsGroup: 1001
          runAsUser: 1001
          privileged: false
          allowPrivilegeEscalation: false
          readOnlyRootFilesystem: true
```
and if there are volumes the following in the pod level
```
      securityContext:
        fsGroup: 1001
```

an emptyDir volume will be needed fo the db